 Dialer: InCallUI: Enable Sustained Performance Mode

We should make sure that thermals are stable during
active calls, so we should enable Sustained Performance
Mode for the InCallActivity.

As a side effect this will also save power during active
calls.

diff --git a/InCallUI/src/com/android/incallui/InCallActivity.java b/InCallUI/src/com/android/incallui/InCallActivity.java
index 5f0f36f..bb89010 100644
--- a/InCallUI/src/com/android/incallui/InCallActivity.java
+++ b/InCallUI/src/com/android/incallui/InCallActivity.java
@@ -191,6 +191,9 @@ protected void onCreate(Bundle icicle) {
                 | WindowManager.LayoutParams.FLAG_IGNORE_CHEEK_PRESSES;
 
         getWindow().addFlags(flags);
+
+        // Enable Sustained Performance Mode
+        getWindow().setSustainedPerformanceMode(true);
 
         // Setup action bar for the conference call manager.
         requestWindowFeature(Window.FEATURE_ACTION_BAR_OVERLAY);
